purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -788,9 +791,26 @@ class Purchases internal constructor(
     }
     // endregion
 
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    fun redeemWebPurchase(webRedemptionLink: DeepLink.WebRedemptionLink, listener: RedeemWebResultListener) {
+        purchasesOrchestrator.redeemWebPurchase(webRedemptionLink, listener)
+    }
+
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    sealed class DeepLink {
+        class WebRedemptionLink internal constructor(internal val redemptionToken: String) : DeepLink()
+    }
+
     // region Static
     companion object {
 
+        @ExperimentalPreviewRevenueCatPurchasesAPI
+        @JvmStatic
+        fun parseIntent(intent: Intent?): DeepLink? {
```

Comment:
I think `Intent` can be not-nullable here, unless I'm missing something? From an API perspective, it communicates that `null` is valid input, but I don't think it is? (It will never parse into a `DeepLink`.)

Reply:
yeah... I did it for simplicity purposes so devs don't need to check for a null intent... But I agree, it doesn't make sense. Will change it, thanks!